python/ray/data/dataset.py

```diff
@@ -2348,7 +2349,9 @@ def iter_batches(
         Args:
             prefetch_blocks: The number of blocks to prefetch ahead of the
                 current block during the scan.
-            batch_size: Record batch size, or None to let the system pick.
+            batch_size: Request a specific batch size, or None to use entire blocks
+                as batches (blocks may contain different number of rows).
+                Defaults to a system-chosen batch size.
```

Comment:
```suggestion
                Defaults to 4096.
```

Let's be specific about the chosen batch size, here and elsewhere.